test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -339,6 +339,42 @@ configure_database(scoped_session &session)
     }
 }
 
+// Take a backup of the provided database and then delete it. This backup will be used in the
+// next loop as a source directory.
+static void
```

Comment:
I think we can probably name this better, right now this function may as well be called "burn_cpu_cycles" or something.

Reply:
cycles unburned in https://github.com/wiredtiger/wiredtiger/pull/11757/commits/8304089cb2b23290c3dd95e99f50670637cb7a07